man/systemctl.xml

```diff
@@ -638,10 +638,13 @@
           <term><command>list-units <optional><replaceable>PATTERN</replaceable>...</optional></command></term>
 
           <listitem>
-            <para>List known units (subject to limitations specified
-            with <option>-t</option>). If one or more
-            <replaceable>PATTERN</replaceable>s are specified, only
-            units matching one of them are shown.</para>
+            <para>List active units, i.e. units that systemd has loaded because they were either
+            referenced directly or through a dependency. By default only units which are active or
+            have pending job are shown; this can be changed with option <option>--all</option>. If
```

Comment:
actually, by default all units which are active, have a pending job or have failed are shown. The latter is pretty important to mention I think